seed/lib/mappings/data/pm-mapping.json

```diff
@@ -48,7 +57,7 @@
     "display_name": "Building Count",
     "to_field": "building_count",
     "to_table_name": "PropertyState",
-    "from_field": "Number of Buildings",
+    "from_field": "How Many Buildings?",
```

Comment:
`Number of Buildings` is represented as a string, `"One"`, while `How Many Buildings?" is represented as an integer, `1`

Reply:
I've seen this in ESPM files. Is that what you are saying--that there are two fields, one is numeric and the other is string?

Reply:
Correct, both `Number of Buildings` and `How Many Buildings?` appear in the ESPM files. One is a string, the other is an integer.

Reply:
Yeah, we need both of these, crazy right?!

Reply:
So are we trying to keep both of these or only import "how many buildings"?